.travis/deploy.sh

```diff
@@ -28,6 +28,9 @@ fi
 message BUILDING EXECUTE DOCKER IMAGE
 docker build -f dockerfiles/execute/Dockerfile . -t makerdao/vdb-execute:$TAG
 
+message BUILDING EXTRACT-DIFFS DOCKER IMAGE
+docker build -f dockerfiles/extract_diffs/Dockerfile . -t makerdao/vdb-extract-diffs:$TAG
```

Comment:
Since this is moving over here, do we want a corresponding PR to remove it from vdb?

Reply:
👍 I think that https://github.com/makerdao/vulcanizedb/pull/119 should take care of this.